lib/import/Importer.js

```diff
@@ -76,7 +76,15 @@ export function importBpmnDiagram(diagram, definitions, bpmnDiagram) {
     }
 
     var mainDiagram = bpmnDiagram || definitions.diagrams[0];
-    canvas.setActivePlane(mainDiagram.plane.bpmnElement.id);
+
+    var rootId = mainDiagram.plane.bpmnElement.id;
+
+    // we do need to account for different ways we create root elements
+    // each nested imported <root> do have the `_plane` suffix, while
+    // the root <root> is found under the business object ID
+    canvas.setRootElement(
+      canvas.findRoot(rootId + '_plane') || canvas.findRoot(rootId)
```

Comment:
Here we could use the `planeId` utility

Reply:
Unfortunately we cannot use the utility here. How to find the root depends on whether we import the sub-process as the root of the diagram (it is called `subprocess(nosuffix)` then or as a nested sub-process plane, called `subprocess_plane` then).

Reply:
=> It is safer to look it up searching for plane first, and actual element next.

Reply:
Makes sense with the current semantic of `open`. 
We should revisit this when we tackle https://github.com/bpmn-io/bpmn-js/issues/1546, as "subProcess is root only" should not exist after that